code/obj/machinery/plantpot.dm

```diff
@@ -701,7 +701,8 @@ TYPEINFO(/obj/machinery/plantpot)
 				boutput(user, "<span class='alert'>There is nothing in [W] to pour!</span>")
 				return
 			else
-				user.visible_message("<span class='notice'>[user] pours [W:amount_per_transfer_from_this] units of [W]'s contents into [src].</span>")
+				//corrects the amount of reagents shown to have been used when pouring into a tray
+				user.visible_message("<span class='notice'>[user] pours [min(W:amount_per_transfer_from_this,W.reagents.total_volume)] units of [W]'s contents into [src].</span>")
```

Comment:
why not just use the return value from `trans_to()`?

Reply:
I was looking at how chem/beakers did it and used that
[https://github.com/goonstation/goonstation/blob/262b3faab2014af32ed03bed6df6bcc916328722/code/modules/chemistry/Chemistry-Tools.dm#L201C1-L201C1](url)

Reply:
because that code block isn't for transferring, it's for splashing the beaker on something

Reply:
thanks